### PR TITLE
Hb revamp pg

### PR DESCRIPTION
**Pull request by pdgurav**
#### What does this PR do?
Adding New Components for Generic key value targeting UI

#### What are the relevant issues?
Currently dist folder is also pushed fro building branch on stage, later on will be removed

### CODE REVIEW

File: src/js/components/CheckBox/stories/CheckBoxMaterial.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
```
Comment (amitmishrg) [inline src/js/components/CheckBox/stories/CheckBoxMaterial.js:1]: Remove this file. Don’t need to add extra material storybook, we are making it the default theme.

File: .gitignore [hunk 2]
```diff
@@ -12,7 +12,7 @@ coverage
 package-lock.json
 
 # production
-dist
+# dist
```
Comment (amitmishrg) [inline .gitignore:15]: Remove dist folder before merging to master

File: src/js/components/CheckBox/stories/CheckBoxMaterial.js (hunk 1 above)
Comment (pdgurav) [reply src/js/components/CheckBox/stories/CheckBoxMaterial.js]: Removed

File: .gitignore (hunk 2 above)
Comment (pdgurav) [reply .gitignore:15]: Yes

File: src/js/components/Select/stories/components/SearchBorderBox.js [hunk 3]
```diff
@@ -69,7 +69,7 @@ const defaultStyle = css`
     &:after {
       left: 0;
       width: 100%;
-      background: ${normalizeColor('brand', props.theme)};
+      background: ${normalizeColor('#3367D6', props.theme)};
```
Comment (pdgurav) [inline src/js/components/Select/stories/components/SearchBorderBox.js:72]: remove hardcoded color

File: src/js/components/Select/stories/Multiple.js [hunk 4]
```diff
@@ -2,22 +2,25 @@ import React, { useState } from 'react';
 import { storiesOf } from '@storybook/react';
 
 import { Box, MnetUIBase, Select } from 'mnet-ui-base';
-import { mnet } from 'mnet-ui-base/themes';
+import { hb } from 'mnet-ui-base-theme-hb';
 
-const options = ['one', 'two'];
+const options = ['one', 'two', 'Three', 'Four', 'Five', 'Eighteen'];
```
Comment (pdgurav) [inline src/js/components/Select/stories/Multiple.js:7]: No dependency on theme

File: src/js/components/TextInput/StyledTextInput.js [hunk 5]
```diff
@@ -35,7 +35,7 @@ const StyledTextInput = styled.input`
     props.icon &&
     (props.reverse
       ? `padding-right: ${props.theme.global.edgeSize.large};`
-      : `padding-left: ${props.theme.global.edgeSize.large};`)}
+      : `padding-left: ${props.theme.global.edgeSize.xlarge};`)}
```
Comment (pdgurav) [inline src/js/components/TextInput/StyledTextInput.js:38]: move to extend

File: src/js/components/TextInput/StyledTextInput.js [hunk 6]
```diff
@@ -49,6 +49,13 @@ const StyledTextInput = styled.input`
       props.theme.textInput.disabled && props.theme.textInput.disabled.opacity,
     )}
   ${props => props.theme.textInput && props.theme.textInput.extend};
+  box-shadow: none;
+  ${props => !props.plain && `border: ${props.theme.textInput.border}`}
+  border-bottom-width: 2px;
+  ${props => props.focus && props.theme.textInput.focus}
+  ${props => props.error && props.theme.textInput.error.extend}
+  transition: width 0.2s ease 0s, background 0.2s ease 0s, border 0.2s ease 0s;
```
Comment (pdgurav) [inline src/js/components/TextInput/StyledTextInput.js:57]: move to extend

File: src/js/components/Tooltip/Tooltip.js [hunk 7]
```diff
@@ -10,13 +10,24 @@ import { Drop } from '../Drop';
...
+      children,
+      message,
+      body = false,
+      position = 'right',
+      title,
+      showArrow = true,
+      ...rest
+    },
+    ref,
+  ) => {
```
Comment (pdgurav) [inline src/js/components/Tooltip/Tooltip.js:24]: Param to change fade out/ retain tooltip on hover

**Pull request closed without merging**